extensions/amp-analytics/0.1/instrumentation.js

```diff
@@ -276,29 +284,50 @@ export class InstrumentationService {
   }
 
   /**
+   * @param {!Element} el
+   * @private
+   * @return {Object}
+   */
+  extractAnalyticsVariables_(el) {
+    const dataSet = el.dataset;
+    const analyticsVariables = {};
+    for (const dataKey in dataSet) {
+      if (dataKey.indexOf(vars) === 0) {
+        const variableName = dataKey.toLowerCase().replace(vars,'');
```

Comment:
I think the latest example brings up an issue with this line. Vars have been case sensitive till now. If we lowercase the key, the casing gets lost. Can we avoid doing toLowerCase?